source/test/TestApp/PlayFabApiTest.cpp

```diff
@@ -737,7 +737,10 @@ namespace PlayFabUnit
         AddTest("LoginWithAdvertisingId", &PlayFabApiTest::LoginWithAdvertisingId);
         AddTest("UserDataApi", &PlayFabApiTest::UserDataApi);
         AddTest("PlayerStatisticsApi", &PlayFabApiTest::PlayerStatisticsApi);
-        AddTest("GetServerTime", &PlayFabApiTest::GetServerTime);
+        // BUG 41168 - GetServerTime relies on the client device configurations,
+        // leading to a lot of build failures especially when moving to other platforms
+        // when we have the time, we can revisit this test and replace it with something more reliable.
+        //AddTest("GetServerTime", &PlayFabApiTest::GetServerTime);
```

Comment:
I disagree.
This test is fundamentally necessary to determine if we're [de]serializing timestamps correctly.
I think if we have one platform acting up, we can hide the test from that platform as a temporary measure, but we shouldn't remove the test from working platforms.

Reply:
Timestamps are demonstrably very difficult to get right, in that we've already sunk more than a dev-week into it, just for XPlat, and despite vast improvements, still having issues.
It's important to me that we don't lose the progress we've made.

Reply:
We aren't throwing away any progress. The existence of the bug will track it.

For your specific issue on [de]serializing timestamps. We pass that test before checking against the system clock don't we? If we get a valid time back (doesn't matter what time it is w.r.t this client) it means we [de]serialized the time properly.